internal/gatewayapi/securitypolicy.go

```diff
@@ -366,20 +366,24 @@ func (t *Translator) translateSecurityPolicyForRoute(
 	}
 
 	if policy.Spec.OIDC != nil {
-		if oidc, err = t.buildOIDC(policy, resources); err != nil {
+		if oidc, err = t.buildOIDC(
+			irConfigName(policy),
```

Comment:
nit: curious why this passed for `buildOIDC` but computed for `buildBasicAuth`

Reply:
I missed that :-(  

It shouldn't be needed anymore.

https://github.com/envoyproxy/gateway/pull/2992 @arkodg